tests/filesystem/test_filesystem.py

```diff
@@ -10,40 +11,42 @@
 )
 
 
-def test_file_mimetype(png_image, jpg_image):
+def test_file_mimetype(png_image: pathlib.Path, jpg_image: pathlib.Path):
     assert get_file_mimetype(png_image) == "image/png"
     assert get_file_mimetype(jpg_image) == "image/jpeg"
 
 
-def test_content_mimetype(png_image, jpg_image):
+def test_content_mimetype(png_image: pathlib.Path, jpg_image: pathlib.Path):
     with open(png_image, "rb") as fh:
         assert get_content_mimetype(fh.read(64)) == "image/png"
 
     with open(jpg_image, "rb") as fh:
         assert get_content_mimetype(fh.read(64)) == "image/jpeg"
 
 
-def test_content_mimetype_fallback(monkeypatch, undecodable_byte_stream):
+def test_content_mimetype_fallback(
+    monkeypatch: pytest.MonkeyPatch, undecodable_byte_stream: bytes
+):
     # use raw function first to test actual code
     assert get_content_mimetype(undecodable_byte_stream) == "application/octet-stream"
 
     # mock then so we keep coverage on systems where magic works
-    def raising_magic(*args, **kwargs):  # noqa: ARG001
+    def raising_magic(*_: Any, **__: Any):
```

Comment:
Use conventional `*args` and `**kwargs` and add ruff ignore for unused params. clearer